wiki/en/Running-a-Server.md

```diff
@@ -136,63 +116,62 @@ Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s
+When running as a registered Server this displays the Server's name, city and country so that other users can easily identify it in the Directory listing.
 
 ### Chat Welcome Message
 
-The text entered here appears to all users when they join the server (the chat window will open automatically for them).
+The text entered here appears to all users when they join the Server (the chat window will open automatically for them).
 
 ## Options
 
 
-<figure><img src="{% include img/en-screenshots/server-window-options.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of Jamulus server window options"></figure>
+<figure><img src="{% include img/en-screenshots/server-window-options.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of Jamulus Server window options"></figure>
 
 ### Recording Directory
 
-This sets the path to where server's recording will be stored. With this path set, the "Enable Jam Recoder" function in the Server Setup tab will make recording start once the first person connects to the server, and stops when the last person leaves. Use the "New Recording" button to create a new sub-directory in which the recordings will be stored from then on. Note that Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
+This sets the path to where Server's recording will be stored. With this path set, the "Enable Jam Recoder" function in the Server Setup tab will make recording start once the first person connects to the Server, and stops when the last person leaves. Use the "New Recording" button to create a new sub-directory in which the recordings will be stored from then on. Note that Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
 
-**Note**: When your server is recording, clients will display a message that recording is on.
+**Note**: When your Server is recording, clients will display a message that recording is on.
 
-### Delay panning
+### Custom Directory Server Address
 
-This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal "volume" panning. Note that to achieve the effect, all clients should be set to 'mono in / stereo out' (or to 'stereo', but only if transmitting a stereo signal).
+Leave this field empty unless you need to list your Server on a [Custom Directory](#3-directory)
 
-### Custom Directory Server Address
+### Server List Filename
 
-Leave this field empty unless you need to list your server on a [private Directory](#3-directory)
+Leave this field empty unless you need to list your Server on a [Custom Directory](#3-directory)
```

Comment:
No!  This is only used when running _as a directory_.

Reply:
So "Leave this field empty unless you want to run your Server as a [Custom Directory](#3-directory)"? (I have to admit I am confused by this)

Reply:
Running as a server - not registered
Running as a server - registered on a built in directory
Running as a server - registered on a custom directory

None of those need to store registered servers' details in the file because no other servers are registering with them.

----

Running as a directory

Only when running as a directory (whether or not you're a custom directory -- all of mine and I think all of Volkers -- use this option) do you need to store details of servers registering with your directory.  That's what this field is used for.